osbenchmark/benchmark.py

```diff
@@ -697,6 +698,8 @@ def dispatch_list(cfg):
         test_execution_orchestrator.list_pipelines()
     elif what == "test_executions":
         metrics.list_test_executions(cfg)
+    elif what == "aggregated_results":
```

Comment:
Since you've introduced a new method called `list_aggregations()`, how do you feel about using `aggregations` here as opposed to `aggregated_results`? Aggregations would be more simple for users to type out. But on second thought, aggregations might be confused with OpenSearch aggregations. I'm open to either

Reply:
Personally, I think `aggregated_results` is more explicit and clear about what it's referring to. But we could make it so either `aggregations` or `aggregated_results` triggers the same function maybe?

Reply:
`aggregated_results` is more explicit and we can go forth with that